26 Update LiveviewData

Kaszz opened the pull request:
Closes #26 

Updated the LiveviewData component.
It now utilizes the static batch data which is refreshed when the machine reaches the EXECUTE state.

![](https://i.imgur.com/lnIOlik.png)

Commit:
Update liveviewData

Commit:
Update batch data when in Execute state

Kechr commented:
![Screenshot_20210520_105726](https://user-images.githubusercontent.com/55458479/118950135-34dc9a80-b95a-11eb-9c93-acdbf08dc276.png)

Skal der gøres noget ved det her? (Det er ikke skalerbart, fungerer kun pä en skærm der > 1500px)

LeaChr96 commented:
> Skal der gøres noget ved det her? (Det er ikke skalerbart, fungerer kun pä en skærm der > 1500px)

Kinda yikes, hvis det scaler så dårligt.

Commit:
Code reuse and responsiveness

Used the LiveDataItem and LiveDataProgress components to reuse the different elements.

Kaszz commented:
Used the LiveDataItem and LiveDataProgress components to reuse the different elements.
Also made it more responsive.

![](https://s3.gifyu.com/images/asdasdasdasd23f764ea8ec33681.gif)

Commit:
Remove comment
